python/ray/external_storage.py

```diff
@@ -133,15 +133,11 @@ def _write_multiple_objects(self, f: IO, object_refs: List[ObjectRef],
             # 24 bytes to store owner address, metadata, and buffer lengths.
             assert self.HEADER_LENGTH + address_len + metadata_len + buf_len \
                 == len(payload)
-            # TODO (yic): Considering add retry here to avoid transient issue
-            try:
-                written_bytes = f.write(payload)
-                url_with_offset = create_url_with_offset(
-                    url=url, offset=offset, size=written_bytes)
-                keys.append(url_with_offset.encode())
-                offset = f.tell()
-            except IOError:
-                return keys
+            written_bytes = f.write(payload)
```

Comment:
@iycheng @rkooo567 removed this block since it was hiding the exception. Note that it seems we currently retry spilling errors forever, I filed another issue to track this: https://github.com/ray-project/ray/issues/17017